Sell/Basket/Repository/BasketError.swift

```diff
@@ -64,7 +64,9 @@ extension Array where Element == BasketError {
         } else if count == 1, let error = first {
             return error.message
         } else {
-            return Set(map { $0.message }).joined(separator: "\n")
+            return Set(map { $0.message })
+                .sorted()
```

Comment:
May I know why we need sorted() here please?

Reply:
In the unit test, the `Set` hashes the errors regardless of the sequence for some reason sometimes, so adding a sorted() here for comparison.

Reply:
In this case could we just use `Array`? 
It's because `Set` is a collection with no defined ordering, but if we need to sort it for unit test usage. Then I think maybe we could just use Array here.

Reply:
I'm afraid it's not possible as it doesn't make sense to show duplicated error messages in one string.

Reply:
Thank you, so do you mean that `graphQLError.message` will return same string for different BasketError cases?

Reply:
Possibly